ansible/roles/jupyterhub/templates/jupyterhub_config_base.py.j2

```diff
@@ -100,6 +100,10 @@ c.JupyterHub.redirect_to_server = False
 base_url = os.environ.get('JUPYTERHUB_BASE_URL') or ''
 c.JupyterHub.base_url = base_url
 
+# Shutdown servers on logout
+shutdown_on_logout = os.environ.get('JUPYTERHUB_SHUTDOWN_ON_LOGOUT') or True
```

Comment:
I think the environment values are read as string so in this case we have to parse that value as a boolean

Reply:
@netoisc I checked the type after fetching this default value from ansible and the env var and it’s passed as a string (for example if the user changes this setting to False. Should we cast it to a Boolean?